andela/wg-millennium-falcon
# #157731570 Reset cache after muscle is deleted

#### What does this PR do?
Reset the cache when a muscle is deleted

#### Description of Task to be completed?
When one muscle or more are deleted, the cache is reset so that that muscle is not displayed in any other view where it was being displayed.

#### How should this be manually tested?
After logging on, navigate to the `muscles` page and delete a muscle(s). After navigate to any other page such as `muscle overview` page, here you will notice that the deleted muscle(s) will not be displayed anymore.

#### Any background context you want to provide?
Prior to this, when a muscle(s) was deleted from the `muscles` page, this deletion would not be reflected on the other pages where that muscle(s) was appearing.

#### What are the relevant pivotal tracker stories?
[#157731570](https://www.pivotaltracker.com/story/show/157731570)

[Fixes #157731570]
